python-pro-data-1/podmineny-vyber/excs/ceska-jmena/assets/jmena.csv

```diff
@@ -1,4 +1,4 @@
-jméno,četnost,věk,pohlaví,původ
+jméno,četnost,průměrný_věk,pohlaví,původ
```

Comment:
```suggestion
jmeno,cetnost,prumerny_vek,pohlavi,puvod
```
Já myslel, že už jsme několikrát tu diakritiku z dat vyhazovali. Strašně to zdržuje.